oaknational/Oak-Web-Application
# fix(about us): link wraps incorrectly

## Description

about-us-summary-card links were wrapping incorrectly:
![image](https://user-images.githubusercontent.com/48293828/203613417-a357ffeb-bfd0-4c1f-859e-cedb8b20052a.png)

the fix is to move the links on to their own row, so that they don't wrap for screen width > 750px:

![image](https://user-images.githubusercontent.com/48293828/203614213-f6298ce8-53e7-4340-8119-f25ebd85aaca.png)


## Issue(s)

Fixes #909 

## How to test

1. Go to "https://www.thenational.academy/about-us/who-we-are"
2. change screen size between 700px and 1600px 
4. You should see that the links never wrap.

## Screenshots

How it used to look (delete if n/a):
![image](https://user-images.githubusercontent.com/48293828/203613417-a357ffeb-bfd0-4c1f-859e-cedb8b20052a.png)

How it should now look:
![image](https://user-images.githubusercontent.com/48293828/203614213-f6298ce8-53e7-4340-8119-f25ebd85aaca.png)

## Checklist
